api/feature_links_api.py

```diff
@@ -41,3 +41,20 @@ def do_get(self, **kwargs):
       }
     else:
       self.abort(400, msg='Missing feature_id')
+
+
+class FeatureLinksSummaryAPI(basehandlers.APIHandler):
+  """FeatureLinksSummaryAPI will return all links to the client."""
+
+  @permissions.require_admin_site
+  def do_get(self, **kwargs):
+    feature_links = FeatureLinks.query().fetch(
+        projection=[
+            FeatureLinks.feature_ids,
+            FeatureLinks.url,
+            FeatureLinks.type,
+            FeatureLinks.is_error,
+            FeatureLinks.http_error_code,
+        ]
+    )
+    return [item.to_dict() for item in feature_links]
```

Comment:
Why not implement the stats calculation in python?

Doing the stats on the server side would make it much more scalable because it would not  send every FeatureLink in the JSON response.

Also, I generally find python easier to read, test, and maintain than JS.